static/bidder-info/adipolo.yaml

```diff
@@ -0,0 +1,21 @@
+endpoint: "http://rtb.adipolo.live?pid={{.SourceId}}&host={{.Host}}&pbs=1"
+maintainer:
+  email: "smart@adipolo.com"
```

Comment:
We sent an email to this address to confirm it is reachable. Please respond to the email with "received" to confirm.